CentralComputing/Pod.cpp

```diff
@@ -26,6 +26,16 @@ void Pod::logic_loop() {
       command->value = 0;
     }
 
+    std::ofstream out("/sys/class/gpio/gpio37/direction"); //How often to toggle the GPIO?
+    out<<"out";
+    out.close();
+    out.open("/sys/class/gpio/gpio37/value");
+    out<<"1";
+    out.close();
+    out.open("/sys/class/gpio37/value");
```

Comment:
It looks like this should work! We just need to wrap it into a Utils call, and then test to see if it actually does work

Reply:
You want to put the function that handles the GPIO stuff into Utils?

Reply:
Yes, wherever you think is appropriate. 
Should be the sort of thing where you can call:
`PLACEHOLDER::gpio_toggle(37)`
or 
`PLACEHOLDER::gpio_on(37)`  or `PLACEHOLDER::gpio_off(37)`